third_party/ssc/CMakeLists.txt

```diff
@@ -140,12 +140,9 @@ endfunction()
 # turn off examples, tests and install for jsoncpp
 set(JSONCPP_WITH_EXAMPLE 0)
 set(JSONCPP_WITH_TESTS 0)
-macro (install)
-endmacro ()
-add_subdirectory(jsoncpp)
-macro (install)
-  _install(${ARGV})
-endmacro(install)
+
+# Skip install() commands for jsoncpp
+add_subdirectory(jsoncpp EXCLUDE_FROM_ALL)
```

Comment:
The fix itself looks pretty simple, remove the override of the `install` command.

(An alternative to EXCLUDE_FROM_ALL is to modify the jsoncpp/CMakeLists.txt and comment out the install calls)